tests/foreman/ui/test_contentview.py

```diff
@@ -540,11 +538,8 @@ def test_positive_create_composite(self):
             self.content_views.publish(cv_name2)
             self.content_views.create(composite_name, is_composite=True)
             session.nav.go_to_select_org(org.name)
-            session.nav.go_to_content_views()
             self.content_views.add_remove_cv(
                 composite_name, [cv_name1, cv_name2])
-            self.assertIsNotNone(self.content_views.wait_until_element(
```

Comment:
1) It is redundant verification as we have proper validation in `add_remove_cv` method itself
2) It is invalid verification as alert disappears when we switch between sub-tabs